api.yml

```diff
@@ -17,13 +17,38 @@ tags:
   - name: public
     description: Public APIs facing the Holder and Verifier.
 paths:
-  /credential:
+  /credentials:
```

Comment:
I would have thought this would be a sibling (e.g., issue) of createAndIssue rather than the parent. Could you please help me understand the rationale?

Reply:
hmm:

1.)
POST /credentials 
POST /credentials/createAndIssue

vs:

2.)
POST /credentials/issue 
(maybe later) POST /credentials/compose
POST /credentials/composeAndIssue

Which do people prefer?

Reply:
This is no longer accurate. can I resolve this?

Reply:
they are both siblings now, I'm going to resolve. please reopen / request changes if you disagree.